deploy/ray-operator/api/v1alpha1/raycluster_types.go

```diff
@@ -56,8 +56,8 @@ type RayClusterImage struct {
 }
 
 type Extension struct {
-	// the size of this pod group
-	NumNodes int32 `json:"numNodes"`
+	// the replicas of this pod group
+	Replicas int32 `json:"replicas"`
```

Comment:
Prefer to use *int32. In the future the replicas can be nil, a mutating webhook can be used to set defaults for the resource.

Ref https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L124

Reply:
Sure, it's more reasonable